src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -220,26 +223,14 @@ public function getEmails(): array
 
     /**
      * Deletes all stored emails.
+     *
      * @internal
      */
     public function clearEmails(): void
     {
         $this->emails = [];
     }
 
-    /**
-     * @internal
-     */
-    public function getComponent(string $name): object|null
```

Comment:
I just want to confirm if this deletion was intentional.

Reply:
It was, it was marked as internal already so it's removed now since we don't use it internally.